[Doc] Add usability success to the doc (#223)
Huy-Ngo/acanban: docs/source/appendix/glossary.rst

.. glossary::

   Accessibility

      Accessibility_ means that the technology is developed so that people
      with disabilities can use them.  It also benefits people without
      disabilities, such as people using devices with small screens
      or using slow Internet connection.

>       Our system is accessible more in the latter sense.

Reviewer: The glossary need not to involve the context of the system:

```suggestion
      In this document, *accessibility* usually refers to the latter sense.
```
